sig-cluster-lifecycle/annual-report-2023.md

```diff
@@ -20,16 +34,46 @@
       Please raise an issue in kubernetes/community, if the KEP metadata is correct but the generated list is incorrect.
 -->
 
+As established by our self-assessment mentioned above, the most common areas where we need help are:
+- Contributing to code
+- Reviewing code
```

Comment:
Are there any specific onboarding guides or list of issues/PRs for new contributors?

Reply:
all the listed subprojects have the contributing.md, but only kubeadm has some extra info in it for onboarding new contributors like instructional videos.  it's up to the subprojects to understand that if they need contributors, they need to add better instructions. as a governance unit we can only suggest to them how to improve that.

also, in reality it feels like the quality of contributor onboarding facilities are not a big factor. if someone has the motivation (e.g. being paid)  to contribute they will reach out and find a way.

Reply:
> kubeadm has some extra info in it for onboarding new contributors like instructional videos

Amazing. 👍🏽 

> also, in reality it feels like the quality of contributor onboarding facilities are not a big factor. if someone has the motivation (e.g. being paid) to contribute they will reach out and find a way.

The last sentence is very true. But in addition, onboarding guides help in easing the friction and barriers of contributions and that's why we should have those. Think what if Kubernetes did not contributor onboarding documentation in k/community, maintainers would have needed to answer the same questions many many times more.